libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -240,6 +241,88 @@ public RestClient peekRestClient(Account acc) {
         }
     }
 
+    /**
+     * One time upgrade steps from older versions to Mobile SDK 8.2. Only for internal use!
+     *
+     * @deprecated Will be removed in Mobile SDK 10.0.
+     */
+    public static synchronized void upgradeTo8Dot2() {
```

Comment:
Migration steps for account data stored in `AccountManager`. The key here is to NOT call any `UserAccountManager` APIs or anything else that will call `encrypt` or `decrypt` behind the scenes.